tests/UI/campaigns/functional/FO/03_userAccount/05_getGDPRDataInCSV.js

```diff
@@ -0,0 +1,968 @@
+
+        await dashboardPage.goToSubMenu(
+          page,
+          dashboardPage.customersParentLink,
+          dashboardPage.customersLink,
+        );
+
+        await customersPage.closeSfToolBar(page);
+
+        const pageTitle = await customersPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(customersPage.pageTitle);
+      });
+
+      it(`should filter by customer first name '${customerData.firstName}'`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'filterByCustomerFirstName1', baseContext);
+
+        await customersPage.filterCustomers(page, 'input', 'firstname', customerData.firstName);
+
+        const numberOfCustomersAfterFilter = await customersPage.getNumberOfElementInGrid(page);
+        await expect(numberOfCustomersAfterFilter).to.equal(1);
+      });
+
+      it('should get creation account date', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'getRegistrationDate', baseContext);
+
+        const registration = await customersPage.getTextColumnFromTableCustomers(page, 1, 'date_add');
+
+        registrationDate = `${registration.substr(6, 4)}-${registration.substr(0, 2)}-`
+          + `${registration.substr(3, 2)}${registration.substr(11, 8)}`;
+        await expect(registrationDate).to.contains(date.getFullYear());
+      });
+
+      it('should get last visit date', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'getLastVisitDate1', baseContext);
+
+        const lastVisit = await customersPage.getTextColumnFromTableCustomers(page, 1, 'connect');
+        lastVisitDate = `${lastVisit.substr(6, 4)}-${lastVisit.substr(0, 2)}-`
+          + `${lastVisit.substr(3, 2)}${lastVisit.substr(11, 8)}`;
+        await expect(lastVisitDate).to.contains(date.getFullYear());
+      });
+
+      it('should click on view customer', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToViewCustomerPage', baseContext);
+
+        await customersPage.goToViewCustomerPage(page, 1);
+
+        const pageTitle = await viewCustomerPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(viewCustomerPage.pageTitle);
+      });
+
+      it('should get last connections ip address', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'checkLAstConnections', baseContext);
+
+        ipAddress = await viewCustomerPage.getTextColumnFromTableLastConnections(page, 'ip-address');
+        await expect(ipAddress).to.not.be.null;
+      });
+    });
+
+    describe('Check GDPR data in CSV', async () => {
+      it('should check general info', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'checkGeneralInfo', baseContext);
+
+        // To replace isVisible after release of the module psgdpr
+        // const age = await basicHelper.age(customerData.birthDate);
```

Comment:
```suggestion
        // TODO
        // To replace isVisible after release of the module psgdpr
        // const age = await basicHelper.age(customerData.birthDate);
```

Adding maybe a `TODO` to not forget about this